packages/@uppy/companion/src/server/middlewares.js

```diff
@@ -110,9 +107,9 @@ exports.cors = (options = {}) => (req, res, next) => {
   return cors({
     credentials: true,
     origin,
-    methods,
-    allowedHeaders: allowedHeaders.join(','),
-    exposedHeaders: exposedHeaders.join(','),
+    methods: Array.from(allowMethodsMap),
+    allowedHeaders: Array.from(allowHeadersMap).join(','),
```

Comment:
nit: it would be worth defining a `const methods = Array.from(allowMethodsMap)` to avoid the double conversion `Set` -> `Array`.

Reply:
How will it avoid a double conversion?

Reply:
`Array.from` iterates over the set to create a new `Array` object. If you store that array in a variable, you can re-use it, while the current code makes two `Array.from` calls.

Reply:
but the argument is different?
methods: Array.from(allow**Methods**Set),
allowedHeaders: Array.from(allow**Headers**Set)